site/app/libraries/database/DatabaseQueries.php

```diff
@@ -722,6 +722,89 @@ public function getCategoriesIdForThread($thread_id) {
         return $categories_list;
     }
 
+    public function toggleLikes(int $post_id, string $current_user, bool $action): string {
```

Comment:
Add a comment saying what $action means (i.e. what true/false represents). It is not clear when looking at just this code. Or maybe rename $action to a more descriptive name. The code below is a hard to follow which cases handle what.